core/src/main/java/greencity/controller/UserController.java

```diff
@@ -425,23 +425,6 @@ public ResponseEntity<UserVO> findByEmail(@RequestParam String email) {
         return ResponseEntity.status(HttpStatus.OK).body(userService.findByEmail(email));
     }
 
-    /**
-     * Get {@link UserVO} by id.
-     *
-     * @return {@link UserUpdateDto}.
-     * @author Orest Mamchuk
-     */
-    @ApiOperation(value = "Get User by id")
```

Comment:
I think there is no need to delete endpoint. Just rename @ApiOperation in findIdByEmail

Reply:
Then we should close the issue with number 271 as incorrect. These endpoints are not about the same functionality.

1. findById - returning UserVO.class
2. findIdByEmail - returning user's id

In 271 ticket says: **Expected result: Choose a working end-point. Choose a working end-point. In swagger should be one end-point "Get User by id"** 

Anyway for now I rollback these changes and will ask the mentor about how to resolve this problem